pytx3/pytx3/cli/fetch2.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import requests
+import typing as t
+from .. import TE
+from ..dataset import Dataset
+from . import command_base
+
+class Fetch2Command(command_base.Command):
+    """
+    Download content from ThreatExchange to disk.
+
+    Using the CollaborationConfig, identify ThreatPrivacyGroup that
+    corresponds to a single collaboration and fetch related threat updates.
+    """
+
+    @classmethod
+    def init_argparse(cls, ap) -> None:
+        ap.add_argument(
+            "--start-time",
+            type=int,
+            help="Fetch updates that occured on or after this timestamp",
+        )
+        ap.add_argument(
+            "--stop-time",
+            type=int,
+            help="Fetch updates that occured before this timestamp",
+        )
+        ap.add_argument(
+            "--owner",
+            type=int,
+            help="Only fetch updates for indicators that the given app has a descriptor for",
+        )
+        ap.add_argument(
```

Comment:
The argument parsing for "--threat-types" and "--additional-tags" is fixed in a later commit (d3b7431)

Reply:
fixed how? What does `nargs='+'` do?

Reply:
nargs means take in arguments as a list. The "+" means expect at least 1 value if the argument flag is given (i.e. don't allow --threat-types to be an empty list)